wiki/BanchoBot/en.md

```diff
@@ -1,28 +1,154 @@
-<!-- BanchoBot Information -->
 # BanchoBot
 
-![BanchoBot's player card](BanchoBot.jpg "BanchoBot's player card")
+![BanchoBot's user card](BanchoBot.jpg "BanchoBot's user card")
 
-BanchoBot is a bot (i.e. a set of commands with automated responses) programmed by **Echo49** that assists people in chat by displaying certain information and announcing game-related messages.
-It is the host of the [bancho irc](/wiki/Internet_Relay_Chat) and has a user profile with the ID _[#3](https://osu.ppy.sh/u/3)_ (next to peppy who is [#2](https://osu.ppy.sh/u/2)). *It also has its own [twitter profile](https://twitter.com/banchoboat)!*
+BanchoBot is a bot (i.e. a set of commands with automated responses) that assists people in chat by displaying certain information and announcing game-related messages. It was programmed by [Echo49](/users/431) and is the host of the [bancho IRC](/wiki/Internet_Relay_Chat). BanchoBot also has its own [user profile](/users/3) and [Twitter account](https://twitter.com/banchoboat)!
```

Comment:
`user profile` -> `osu! profile`

Reply:
is there a reason why it should be `osu! profile` instead of `user profile`?

Reply:
Not particularly, maybe because BanchoBot isn't actually a user.

Also, I just wanted to be consistent with the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) page with artists who have an `osu! profile` and it's labeled like that.